### PR TITLE
feat: Apply adjusts in preview and content-types

### PR DESCRIPTION
**Pull request by ArthurTriis1**
## What's the purpose of this pull request?

This PR improves the preview feature, redirecting the preview page according to content-type.

Also, apply adjusts in content-types creating a content-type home singleton.

### CODE REVIEW

File: packages/core/src/pages/api/preview.ts [hunk 1]
```diff
@@ -2,6 +2,13 @@ import type { NextApiHandler, NextApiRequest } from 'next'
 
 import { clientCMS } from 'src/server/cms'
 
+const REDIRECTS_BY_CONTENT_TYPE = {
+  home: '/',
+  plp: '/office',
+  search: '/s?q=headphone',
+  pdp: '/apple-magic-mouse/p',
```
Comment (eduardoformiga) [inline packages/core/src/pages/api/preview.ts:9]: what about adding there values in the `faststore.config.js`? we have some LH values there, but i think can be different configs.

File: packages/core/src/pages/api/preview.ts [hunk 2]
```diff
@@ -46,7 +53,7 @@ const handler: NextApiHandler = async (req, res) => {
 
     // Redirect to the path from the fetched locator
     // TODO: apply redirect based on the content
-    res.redirect('/')
+    res.redirect(REDIRECTS_BY_CONTENT_TYPE[locator.contentType] ?? '/')
```
Comment (eduardoformiga) [inline packages/core/src/pages/api/preview.ts:56]: 💯

File: packages/core/cms/faststore/content-types.json [hunk 3]
```diff
@@ -50,14 +50,62 @@
...
+            }
+          }
+        ]
+      }
+    ]
+  },
   {
     "id": "pdp",
     "name": "Product Page",
+    "isSingleton": true,
```
Comment (eduardoformiga) [inline packages/core/cms/faststore/content-types.json:102]: Not sure if it is a singleton

File: packages/core/cms/faststore/content-types.json [hunk 4]
```diff
@@ -50,14 +50,62 @@
...
     "configurationSchemaSets": []
   },
   {
     "id": "plp",
     "name": "Product List Page",
+    "isSingleton": true,
```
Comment (eduardoformiga) [inline packages/core/cms/faststore/content-types.json:108]: same here

**Review by eduardoformiga — APPROVED** [review]
Good Job! ✨  

pre approved!

File: packages/core/cms/faststore/content-types.json (hunk 3 above)
Comment (ArthurTriis1) [reply packages/core/cms/faststore/content-types.json:102]: I think it should be a singleton, but when we add the multiple pdp/plp feature in the future, we can remove it.

**Pull request merged**